app/jobs/document_proofing_job.rb

```diff
@@ -50,9 +49,7 @@ def perform(
       doc_auth_client.post_images(
         front_image: front_image,
         back_image: back_image,
-        selfie_image: nil,
         image_source: image_source(image_metadata),
-        liveness_checking_enabled: false,
```

Comment:
I opened a pul request to manage these args here: https://github.com/18F/identity-idp/pull/7235

tl;dr there is I think we need to leave these in with default values and remove the args form the callers. Once we are confident we have cleaned up all of the callers then we can pull these args.

Reply:
Wait, nevermind this is a caller. This is where we want to remove them. My comment applies to the code below in the `AcuantClient`